File: Day9/isSubsequence_Donghee.cpp
// Donghee Lee
// 6/9/2020

class Solution {
public:
    bool isSubsequence(string t, string s) {
        
        for (int i = 0; i < s.length(); i++) {
            if (t[0] == s[i]) 
                t = t.substr(1);      

            if (t == "") return true;
        }     
        return t == "";
    }
};

File: Day5/isSubsequence_Donghee.py
# Donghee Lee
# 06/09/2020

class Solution:
    def isSubsequence(self, t: str, s: str) -> bool:
        
        for i in range(len(s)) :
            if t == "" : 
                return True
            
            if t[0] == s[i] :
                t = t[1:]
            
        return t == ""
        